### PR TITLE
Issue #2150 Disk validation fails in cmd-config.feature on MacOS

### PR DESCRIPTION
**Pull request by agajdosi**
Also updating Godog to latest release which fixes incomplete failure report when scenario outlines are used.

### CODE REVIEW

File: test/integration/features/cmd-config.feature [hunk 1]
```diff
@@ -213,7 +213,7 @@ user defined options which changes default behaviour of Minishift.
 
   Scenario: Checking that disk-size value was applied
      When executing "minishift ssh -- sudo fdisk -l | grep Disk" succeeds
-     Then stdout should match "Disk \/dev\/sda: 2[4-6]\.?[0-9]{0,2} (GB|GiB)"
+     Then stdout should match "Disk \/dev\/(s|v)da: 2[4-6]\.?[0-9]{0,2} (GB|GiB)"
```
Comment (praveenkumar) [inline test/integration/features/cmd-config.feature]: @agajdosi Can't we use `?da` as regex where we don't care what is the first char? `?da` is what we are using as part of disk check for preflight checks.

Comment (budhrg) [reply test/integration/features/cmd-config.feature]: Yes `?da` is better choice +1

Comment (agajdosi) [reply test/integration/features/cmd-config.feature]: @praveenkumar @budhrg Good point, missed that. Updating now.

File: test/integration/features/cmd-config.feature [hunk 2]
```diff
@@ -213,7 +213,7 @@ user defined options which changes default behaviour of Minishift.
 
   Scenario: Checking that disk-size value was applied
      When executing "minishift ssh -- sudo fdisk -l | grep Disk" succeeds
-     Then stdout should match "Disk \/dev\/sda: 2[4-6]\.?[0-9]{0,2} (GB|GiB)"
+     Then stdout should match "Disk \/dev\/.da: 2[4-6]\.?[0-9]{0,2} (GB|GiB)"
```
Comment (praveenkumar) [inline test/integration/features/cmd-config.feature]: 👍

**Comment by budhrg** [comment]
LGTM :+1:

**Pull request merged**